Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2886,6 +2904,8 @@ private protected virtual void OnDisable()
     [SerializeField] private EntityRecordDictionary m_clientInstancedEntities = new EntityRecordDictionary();
 
     private protected Action m_onMaterialChangedInSceneViewCB = null;
+
+    [SerializeField] internal bool GenerateLODGroup;
```

Comment:
is this needed?

Reply:
No, the feature was rejected, deleted now.